tests/integration/conftest.py

```diff
@@ -26,6 +27,19 @@ def app_name(metadata):
     yield metadata["name"]
 
 
+@fixture(scope="module")
+def run_action(ops_test: OpsTest) -> Callable[[str, str], Awaitable[Any]]:
```

Comment:
Just FYI, type hints on fictional are currently optional, especially the return type since these functions are primarily used by pytest rather than being called elsewhere in the code. I sometimes annotate fixtures in test functions to get the benefit of type checks and autocomplete there

Reply:
By "fictional" you meant "fixture" ?  
I added the return type after @arturo-seijas proposal to do so. In general we do that on a best effort basis ?

Reply:
Yes, I did mean `fixture`. Yes best effort/ when it is worth it. I'm not saying don't do it, just wanted to let you know you that there currently isn't a mandate to do it on all fixtures